.pnpmfile.cjs

```diff
@@ -41,6 +41,36 @@ function fixDeps( pkg ) {
 		pkg.peerDependencies[ '@babel/runtime' ] = '^7';
 	}
 
+	// Missing deps.
+	// https://github.com/WordPress/gutenberg/issues/67864
+	if ( pkg.name === '@wordpress/dataviews' ) {
+		for ( const dep of [
+			'change-case',
+			'colord',
+			'date-fns',
+			'deepmerge',
+			'@emotion/cache',
+			'@emotion/css',
+			'@emotion/react',
+			'@emotion/styled',
+			'@emotion/utils',
+			'fast-deep-equal',
+			'@floating-ui/react-dom',
+			'framer-motion',
+			'highlight-words-core',
+			'is-plain-object',
+			'memize',
+			'react-dom',
+			'@use-gesture/react',
+			'use-memo-one',
+			'uuid',
+			'@wordpress/date',
+			'@wordpress/hooks',
```

Comment:
Are these last two actually needed?

Reply:
I had  copied from the [other PR this list](https://github.com/Automattic/jetpack/pull/41510/files#r1952668188) and while I had checked and updated  the translate functions list, I'll do the same for this one and will remove everything that is not needed!

Reply:
I checked the deps and updated them [here](https://github.com/Automattic/jetpack/pull/41602/commits/940f63a87a89c18fb4ef93c9b8b33613fffb351a), where it seems way fewer deps where enough to build successfully. 

I'd really appreciate a final check here.